tests/schema/models.py

```diff
@@ -68,7 +68,8 @@ class AuthorWithIndexedNameAndBirthday(models.Model):
 
     class Meta:
         apps = new_apps
-        index_together = [["name", "birthday"]]
+        # Added on-the-fly by the test method.
+        # index_together = [["name", "birthday"]]
```

Comment:
`AuthorWithIndexedNameAndBirthday` is used only by a single test method. I'd made it inline in advance.